packages/block-editor/src/utils/dom.js

```diff
@@ -96,11 +96,26 @@ function isElementVisible( element ) {
 		return false;
 	}
 
-	return element.checkVisibility( {
-		opacityProperty: true,
-		contentVisibilityAuto: true,
-		visibilityProperty: true,
-	} );
+	// Older browsers, e.g. Safari may not have `checkVisibility` method.
```

Comment:
Optional, but if we're being specific about the browser, maybe also the version. 

```suggestion
	// Older browsers, e.g. Safari < 17.4 may not support the `checkVisibility` method.
```

Reply:
Yes, good call, thank you! Our future selves will thank us when we inevitably forget which version introduced this method 😄